sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -0,0 +1,107 @@
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using OpenTelemetry;
+using OpenTelemetry.Logs;
+using System;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter
+{
+    /// <summary>
+    /// Extension methods for setting up Azure Monitor in an <see cref="OpenTelemetryBuilder" />.
+    /// </summary>
+    public static class OpenTelemetryBuilderExtensions
+    {
+        /// <summary>
+        /// Configures Azure Monitor Exporter for all signals.
+        /// </summary>
+        /// <param name="builder"><see cref="IOpenTelemetryBuilder"/>.</param>
+        /// <returns>The supplied <see cref="IOpenTelemetryBuilder"/> for chaining calls.</returns>
+        /// <remarks>
+        /// <para>
+        /// This method configures Azure Monitor for use with OpenTelemetry by adding the Azure Monitor exporter for logging,
+        /// distributed tracing, and metrics. It also configures the OpenTelemetry logger to include formatted messages and
+        /// parsed state values.
+        /// </para>
+        /// </remarks>
+        public static IOpenTelemetryBuilder UseAzureMonitorExporter(this IOpenTelemetryBuilder builder)
+        {
+            builder.Services.TryAddSingleton<IConfigureOptions<AzureMonitorExporterOptions>,
+                        DefaultAzureMonitorExporterOptions>();
+            return builder.UseAzureMonitorExporter(o => { });
+        }
+
+        /// <summary>
+        /// Configures Azure Monitor Exporter for logging, distributed tracing, and metrics.
+        /// </summary>
+        /// <param name="builder"><see cref="IOpenTelemetryBuilder"/>.</param>
+        /// <param name="configureAzureMonitor">Callback action for configuring <see cref="AzureMonitorExporterOptions"/>.</param>
+        /// <returns>The supplied <see cref="IOpenTelemetryBuilder"/> for chaining calls.</returns>
+        /// <remarks>
+        /// <para>
+        /// This method configures Azure Monitor for use with OpenTelemetry by adding the Azure Monitor exporter for logging,
+        /// distributed tracing, and metrics. It also configures the OpenTelemetry logger to include formatted messages and
+        /// parsed state values.
+        /// </para>
+        /// </remarks>
+        public static IOpenTelemetryBuilder UseAzureMonitorExporter(this IOpenTelemetryBuilder builder, Action<AzureMonitorExporterOptions> configureAzureMonitor)
+        {
+            if (builder.Services == null)
+            {
+                throw new ArgumentNullException(nameof(builder.Services));
+            }
+
+            if (configureAzureMonitor != null)
+            {
+                builder.Services.Configure(configureAzureMonitor);
+            }
+
+            builder
+                .WithLogging()
+                .WithMetrics(metrics => metrics.AddAzureMonitorMetricExporter())
+                .WithTracing();
```

Comment:
nit: I see that this is only configuring the MetricExporter.
I see below at line 101 that the ExporterRegistrationHostedService is responsible for configuring the other two exporters.

I think we need a comment here at line 67 explaining this strategy.

Reply:
Added code comment.